packages/chip/chip.scss

```diff
@@ -0,0 +1,97 @@
+@charset "UTF-8";
+@use "@fremtind/jkl-core/jkl";
+
+.jkl-chip {
+    --text-color: var(--jkl-color-text-default);
+    --border-color: var(--jkl-color-border-separator);
+    --background-color: transparent;
+    --border-width: #{jkl.rem(1px)};
+
+    @include jkl.compact-density {
+        --padding-block-s: 0;
+        --padding-block-l: #{jkl.$spacing-4};
+        --padding-inline-s: #{jkl.$spacing-8};
+        --padding-inline-l: #{jkl.$spacing-12};
+        --padding-icon-s: #{jkl.$spacing-4};
+        --padding-icon-l: #{jkl.$spacing-8};
+        --gap-s: #{jkl.$spacing-2};
+        --gap-l: #{jkl.$spacing-4};
+    }
+
+    @include jkl.comfortable-density {
+        --padding-block-s: #{jkl.$spacing-4};
+        --padding-block-l: #{jkl.$spacing-8};
+        --padding-inline-s: #{jkl.$spacing-12};
+        --padding-inline-l: #{jkl.$spacing-16};
+        --padding-icon-s: #{jkl.$spacing-8};
+        --padding-icon-l: #{jkl.$spacing-12};
+        --gap-s: #{jkl.$spacing-4};
+        --gap-l: #{jkl.$spacing-8};
+    }
```

Comment:
Vet at Ivar akkurat ba deg om å flytte disse 😅 Men hvis de skal brukes inni klassen må de flyttes ned til linje 46 (før forced-colors-mixinen) eller senere for ikke å få deprecation warnings fra Sass under bygg.

Enda bedre: Jeg har nå fikset bug-en vi fant under Jøkuluka, så du kan nå bruke de nye mixin-ene i toppen av fila (der du hadde de gamle) med klassenavn for scoping 🎉 Altså:

```scss
@include jkl.compact-density(".jkl-chip") {
    /* Variabler her… */
}
```